app/javascript/packs/components/VolunteerMap.jsx

```diff
@@ -53,7 +53,10 @@ const VolunteerMap = ({ center }) => {
   return (
     <Box width="100%" className={classes.mapWrapper}>
       <GoogleMapReact
-        bootstrapURLKeys={{ key: "AIzaSyC91p5MR0B-sProwqXqStywWfAXOxOkcm8" }}
+        bootstrapURLKeys={{
+          key: "AIzaSyC91p5MR0B-sProwqXqStywWfAXOxOkcm8",
+          libraries: "places",
```

Comment:
Does this need to check for placesIsDefined too?

Reply:
Ah good call! After taking a look, the Google Map React library [checks if `google.maps` is defined](https://github.com/google-map-react/google-map-react/blob/master/src/loaders/google_map_loader.js#L37-L40) so we're covered.

Reply:
:+1: